tests/unit_test/callback/pyscript_handler_test.py

```
@@ -1224,6 +1377,205 @@ def test_fetch_collection_data_empty_result():
+            "aadhar": 29383989838930,
+            "pan": "JJ928392JH",
+            "pincode": 538494
+        }
+    }
+    mock_doc1 = MagicMock()
+    mock_doc1.to_mongo.return_value.to_dict.return_value = data1
+
+    mock_doc2 = MagicMock()
+    mock_doc2.to_mongo.return_value.to_dict.return_value = data2
+
+    mock_queryset = [mock_doc1, mock_doc2]
```

Ruff (0.11.9)

Local variable `mock_queryset` is assigned to but never used

Remove assignment to unused variable `mock_queryset`

(F841)